packages/devtools_app/lib/src/inspector/flutter/layout_explorer/flex.dart

```diff
@@ -281,8 +281,8 @@ Widget _visualizeWidthAndHeightWithConstraints({
   );
 }
 
-class StoryOfYourFlexWidget extends StatefulWidget {
-  const StoryOfYourFlexWidget(
+class FlexLayoutExplorerWidget extends StatefulWidget {
```

Comment:
why not just LayoutExplorer? Seems like that would fit better if we support non-flex widgets in the future.

Reply:
I would suggest we add a new LayoutExplorer widget class as well that adds a FlexLayoutExplorerer widget as a child if the widget is flex. This class has a lot of very flex specific logic and is already quite complex to handle everything we care about for flex.

Reply:
we actually have LayoutExplorerTabWidget class in the `inspector_screen_details_tab.dart`, but I'll move them into Layout Explorer directory and nest all the flex-specific implementation in a directory inside the `layout_explorer`